binding/binding.go

```diff
@@ -29,6 +29,13 @@ type Binding interface {
 	Bind(*http.Request, interface{}) error
 }
 
+// BindingBody adds BindBody method to Binding.  BindBody is similar with Bind,
```

Comment:
Please use two spaces, thanks.

Reply:
You mean “one space”, right? mean to replace `...˽Binding.˽˽BindBody˽is...` into `...˽Binding.˽BindBody˽is...` ?